index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
+
+  <body>
+
+    <nav>
+      <ul>
+        <li><a href="#welcome-section"> Welcome </a></li>
+        <li><a href="#works">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+
+    <section class="welcome-section">
+      <h1>Welcome to my portfolio site</h1>
+    </section>
+
+    <section class="works">
+      <h2>Our works</h2>
+      <div class="projects">
+        <div class="project">
+          <div class="project-tile">First Project</div>
+          <a href="https://www.google.com">Link to my project</a>
+        </div>
+        <div class="project">
+          <div class="project-tile">second Project</div>
+          <a href="https://www.google.com">Link to my project</a>
+        </div>
+        <div class="project">
+          <div class="project-tile">third Project</div>
+          <a href="https://www.google.com">Link to my project</a>
+        </div>
+      </div>
+    </section>
+
+    <section class="contact">
+      <h2>Contact Us</h2>
+      <div>
+        <a href="https://github.com/Timowest12">Timo</a>
+        <a id="profile-link" href="https://github.com/Bondok6">kyrillos</a>
+      </div>
+    </section>
```

Comment:
- By adding semantic tags to your document, you provide additional information about that document, which helps with communication. It is also important for search engines (SEO), so **it would be better to use the `<footer>` tag instead of using the `<section>` tag**

- Your implementation using the ` section ` tag 

       <section class="contact">
             <h2> </h2>
             <div>
                  <a> </a>
                  <a> </a>
             </div>
       </section>

- A better implementation using the ` footer ` tag

        <footer class="contact">
             <h2> </h2>
             <div>
                   <a> </a>
                   <a> </a>
             </div>
        </footer>
  
- Your design looks perfect 👌✅💯